products/suse-manager.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SUSE Manager
+category: server-app
+tags: suse
+iconSlug: suse
+permalink: /suse-manager
+versionCommand: zypper info SUSE-Manager-Server-release
+releasePolicyLink: https://www.suse.com/lifecycle
+changelogTemplate: "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/{{'__RELEASE_CYCLE__'}}/"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: General Support
+
+releases:
```

Comment:
Looked at https://github.com/endoflife-date/endoflife.date/pull/5088#pullrequestreview-2161539612 and I have a similar question : where can endoflife.date maintainers find the latest releases and their dates.

In each released notes we have the list of all versions, but only a single date at the document level. Is it what we should use for the latest release date ?

Reply:
Yes, exactly. I wasn't able to find a nicer location where to get these version information.